lib/public/views/Tags/Create/index.js

```diff
@@ -62,6 +62,23 @@ const tagCreationForm = ({ data }) => {
                 data.email = e.target.value;
             },
         }),
+
+        h('label.form-check-label.f4.mt2', { for: 'color' }, 'Color'),
+        h('label.f5.mb2', 'Please fill in a hexadecimal color after the #'),
+        h('input#email.form-control.w-25.mb2', {
+            placeholder: 'Enter the color#...',
+            value: color,
+            minlength: 6,
+            maxlength: 6,
+            oninput: (e) => {
+                if (e.target.value !== '') {
+                    data.color = e.target.value;
+                } else {
+                    data.color = null;
+                }
+            },
+            style: { backgroundColor: `#${color}` },
+        }),
```

Comment:
You can extract a `colorInputComponent` from this. BTW, you actually have a color HTML input (you'll have to remove the `#` from its value actually but it's better to use it